Update syntax for docusaurus (#11)
Topl/Knowledge-Base: docs/Developer/BaaS Portal Onboarding/topl-api-keys.md

createdAt: "2021-04-01T21:43:27.915Z"
updatedAt: "2022-03-29T14:33:28.295Z"
---
All requests submitted through BaaS require an API key. This allows our service to authenticate your account to enforce API quotas.

It is mandatory that you sign up for a free API key which allows for many advantages:

* a **high request rate** and concurrent request limit
* **fast** responses with few retries and timeouts
* useful **metric tracking** for performance tuning and to analyze your customer behavior
* Access to our Genus offerings including the ability to query archived data and advanced logging capabilities (not yet released)

:::info

> You can sign up for API keys by creating a project detailed in the guide below.

Reviewer: ```suggestion
To sign up for API keys, create a project as detailed in the guide below.
```
